docs/antora/modules/ROOT/pages/Library_Dependencies.adoc

```diff
@@ -150,13 +150,48 @@ def compileIvyDeps = Agg(
 )
 ----
 
-It is also possible to use higher version of the same library dependencies already  defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+It is also possible to use higher version of the same library dependencies already defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+
+== Detecting transitive dependencies
+
+To render a tree of dependencies (transitive included) and possible upgrades for a given module, you can run `mill myModule.ivyDepsTree`.
+
+You can also compile your module(s) and examine files such as `ivyDeps.json` and `transitiveIvyDeps.json` in your `out` build's folder for a given module. `ivyDepsTree.log` contains file output of the above ivyDepsTree command too.
 
 == Excluding transitive dependencies
 
 You can use the `.exclude` method on a dependency. It accepts  `organization` and `name` tuples, to be excluded.
 Use the special name `*` to match all ``organization``s or ``name``s.
 
+.Example: Exclude `fansi_2.12` library from transitive dependency set of `pprint`.
+[source,scala]
+----
+def deps = Agg(
+  ivy"com.lihaoyi::pprint:0.5.3".exclude("com.lihaoyi" -> "fansi_2.12")
+)
+----
+
+You can also use `.excludeOrg` or `excludeName`:
+
+
+.Example: Exclude all `com.lihaoyi` libraries from transitive dependency set of `pprint`.
+[source,scala]
+----
+val deps = Agg(ivy"com.lihaoyi::pprint:0.5.3".excludeOrg("com.lihaoyi"))
+----
+
+Note: You can chain multiple exclusions with `exclude`, `excludeOrg`, and `excludeName`.
+
+.Example: Excluding a library (fansi) by name from transitive dependency set of `pprint`.
+[source,scala]
+----
+val deps = Agg(
+  ivy"com.lihaoyi::pprint:0.5.3"
+    .excludeName("fansi_2.12")
+    .excludeName("sourcecode")
+)
```

Comment:
Chaining excludes is useful so I added a little bit on it.

I'm not sure if sourcecode needs a number for the cross-version (sourcecode_2.12)
- https://github.com/com-lihaoyi/PPrint/blob/master/build.sc#L48
- https://mvnrepository.com/artifact/com.lihaoyi/pprint_3/0.8.1